core-web/apps/dotcms-ui/src/app/api/services/dot-favorite-page/dot-favorite-page.service.ts

```diff
@@ -29,7 +29,7 @@ export class DotFavoritePageService {
         url?: string;
         offset?: string;
         sortField?: string;
-        sortOrder?: ESOrderDirection;
+        sortOrder?: ESOrderDirection | string;
```

Comment:
Why?

Reply:
This was required to pass the sortOrder that is recommended in the issue `dotFavoritePage.order ASC, identifier`

Reply:
Are you sure we don't need to add like `sortField` as new property?